runtime/src/main/java/org/corfudb/runtime/view/ReadOptions.java

```diff
@@ -0,0 +1,27 @@
+package org.corfudb.runtime.view;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * When the client issues a read, the request traverses multiple layers and different configurations
+ * can be set for that read requests. This ReadOptions objects enables different configurations per
+ * read request.
+ *
+ * <p>Created by Maithem on 7/9/19.</p>
+ */
+
+@Builder
+@Data
+public class ReadOptions {
+    final boolean ignoreTrim;
```

Comment:
Why there is no default value for ignoreTrim?

Reply:
made it more explicit